rest_framework_swagger/introspectors.py

```diff
@@ -508,10 +532,7 @@ def get_data_type(field):
     # elif isinstance(field, fields.SlugField):
         # return 'string', 'string', # 'slug'
     elif isinstance(field, fields.ChoiceField):
-        first_key = list(field.choices)[0]
-        if isinstance(first_key, int):
-            return 'integer', 'int64'
-        return 'string', 'string'
+        return 'choice', 'choice'
```

Comment:
it was replaced by the `enum` key on params: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameter-object